swupd/format_switch_test.go

```diff
@@ -1,21 +1,27 @@
 package swupd
 
 import (
+	"strings"
 	"testing"
 )
 
-// Format 25 should not support minversions or iterative manifests. Support for
-// these features was added in format 26.
+// Format 25 should not support minversions, iterative manifests, or delta manifests.
+// Support for these features was added in format 26.
```

Comment:
This test is becoming a pretty big test and is testing several different format-switch features. Can we split it out like was done for the contentsize test below?

Reply:
Ya, I can split these out